### PR TITLE
Use RDS companion article

### PR DESCRIPTION
**Pull request by thewilkybarkid**
https://github.com/elifesciences/issues/issues/5421

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
No tests for `ResearchArticleWorkflow`? 😨

File: config/dev.php [hunk 1]
```diff
@@ -17,4 +17,12 @@
         'region' => 'us-east-1',
         'endpoint' => 'http://localhost:4100',
     ],
+    'feature_rds' => false,
+    'rds_articles' => [
+        '30274' => [
+            'date' => '2020-01-29T19:18:00Z',
+            'display' => 'https://hub.stenci.la/elife/30274/main/?elife',
+            'download' => 'https://hub.stenci.la/elife/30274/files/download/article-30274.md',
```
Comment (giorgiosironi) [inline config/dev.php]: `download` should not be necessary?

Comment (thewilkybarkid) [reply config/dev.php]: No, but trying to share the same config as Journal.

File: config/dev.php [hunk 2]
```diff
@@ -17,4 +17,12 @@
         'region' => 'us-east-1',
         'endpoint' => 'http://localhost:4100',
     ],
+    'feature_rds' => false,
```
Comment (giorgiosironi) [inline config/dev.php]: This `config/*.php` files will really need to be moved into pillars for ease of editing by production.

File: src/Search/Workflow/ResearchArticleWorkflow.php [hunk 3]
```diff
@@ -132,7 +135,11 @@ public function index(ArticleVersion $article) : array
             'value' => json_encode($articleObject->dataSets ?? '[]'),
         ];
 
-        $sortDate = $article->getStatusDate();
```
Comment (giorgiosironi) [inline src/Search/Workflow/ResearchArticleWorkflow.php]: seems we don't use other dates (like `versionDate`), so this should be the only entry point for articles

**Pull request merged**